### DISCUSSION TITLE
Why can PC browser display pictures and mobile browser display black screen

### DISCUSSION THREAD

**Discussion by a6965921** [pixijs/pixijs#7307] [Q&A]
Why can PC browser display pictures and every mobile browser display black screen  and There was no error

this is demo
https://codepen.io/a6965921/pen/vYywYZG          

How to solve it

**Comment by bigtimebuddy** [comment]
This works fine for me. You need to be more specific. Which mobile browser?

Also, for general questions or help, please use the discussion board. Thank you.

**Reply by a6965921** [reply]
all mobile browser

**Reply by a6965921** [reply]
every

**Reply by themoonrat** [reply]
It's because the image is too large.
Take a look at https://webglreport.com/ on different devices, especially the "max texture size".

That figure is different depending on different GPU hardware. A decent PC might show 16384x16384, but mobile hardware often has 4096x4096 as the limit. That image you are trying to load is 7265x3681, so beyond the GPU limits. There is no way around this, you just need to load lower resolution images.
